definitions/divorce/json/CaseEventToComplexTypes/CaseEventToComplexTypes-share-a-case-nonprod.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "LiveFrom": "20/01/2021",
+    "ID": "OrganisationPolicy",
+    "CaseEventID": "solicitorCreate",
+    "CaseFieldID": "PetitionerOrganisationPolicy",
+    "ListElementCode": "OrgPolicyCaseAssignedRole",
+    "DefaultValue": "[PETSOLICITOR]",
+    "FieldDisplayOrder": 1,
+    "DisplayContext": "OPTIONAL"
+  },
+  {
+    "LiveFrom": "20/01/2021",
+    "ID": "OrganisationPolicy",
+    "CaseEventID": "solicitorCreate",
+    "CaseFieldID": "PetitionerOrganisationPolicy",
+    "ListElementCode": "Organisation.OrganisationID",
+    "EventElementLabel": "",
```

Comment:
Do we need this element, if it's empty?

Reply:
This was copied from:
https://github.com/hmcts/finrem-ccd-definitions/pull/367/files#diff-82fd3b1b58f7c09b43dc55d10c326b85b48206b774433f5abf6d572a449a3a88R190-R191

Reply:
@Dizrupto @madjava and I were using their release PR as a template to what ours is based off

Reply:
@jvictorcf These fields are optional in the documentation but we noticed that FR added them for these instances. Decided to do same 😜